Libraries/Components/WebView/WebView.android.js

```diff
@@ -127,13 +134,25 @@ var WebView = React.createClass({
       domStorageEnabled = this.props.domStorageEnabledAndroid;
     }
 
+    var htmlProp, urlProp, sourceProp;
+    if(this.props.html) {
+      console.warn('html is deprecated. Use source instead.');
+      htmlProp = this.props.html;
+    } else if(this.props.url) {
+      console.warn('url is deprecated. Use source instead.');
```

Comment:
Since this pattern comes up a lot, perhaps we can come up with a standard way to mark props as deprecated that includes the component info? I think someone mentioned such a thing already exists in React.js?